openapi.yaml

```diff
@@ -10615,6 +10618,300 @@ components:
+          whatsapp:
+            type: icebreaker
+            name: account_update_example_loc_p
+            content:
+              header:
+                location:
+                  longitude: -122.425332
+                  latitude: 37.758056
+                  name: Facebook HQ
+                  address: '1 Hacker Way, Menlo Park, CA 94025'
+              body:
+                - text: John
+                - text: '619704512462'
+        WhatsApp Message Using Image and Button (Quick Reply):
+          to: '61430933333'
+          subject: WhatsApp Message Subject
+          whatsapp:
+            type: icebreaker
+            name: image_quick_reply
+            content:
+              header:
+                image:
+                  link: 'http://placekitten.com/200/300'
+              body:
+                - text: John
+                - text: ACTIVE
+                - text: '619704512462'
+              button:
+                - index: '0'
+                  sub_type: quick_reply
+                  payload: Yes-Button-Payload
+                - index: '1'
+                  sub_type: quick_reply
+                  payload: No-Button-Payload
+        WhatsApp Message Using Location and Link Button (static url):
+          to: '639177176978'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: location_cta_staticurl
+            content:
+              header:
+                location:
+                  longitude: -122.425332
+                  latitude: 37.758056
+                  name: Facebook HQ
+                  address: '1 Hacker Way, Menlo Park, CA 94025'
+              body:
+                - text: John
+                - text: '619704512462'
+        'WhatsApp Message Using Link button (dynamic) ':
+          to: '61430933333'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: new_media_message_rules
+            content:
+              header:
+                image:
+                  link: 'http://placekitten.com/200/300'
+              body:
+                - text: John
+              buttons:
+                - index: '0'
+                  sub_type: url
+                  text: search?q=Whisprr
+        WhatsApp Message Using Link button (static url + call phonenumber):
+          to: '61430933333'
+          subject: message subject
+          whatsapp:
+            type: icebreaker
+            name: hsbc_alert
+            content:
+              body:
+                - text: John
+                - text: Everything is connected!
+                - text: 'https://hsbc.com/renewal'
+                - text: 'https://hsbc.com/terms'
+                - text: 'https://hsbc.com/privacy'
+      properties:
+        type:
+          type: string
+          description: |-
+            The type of WhatsApp message to be sent, e.g. "template".
+
+            Whispir currently only supports type "template".
```

Comment:
If only a set of values can be taken, we can use the JSON schema [enum](https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values) key like follows:

```
type: string
enum:
  - template
```

Reply:
Added enum to type property